weld/easy_ll/tests.rs

```diff
@@ -15,7 +16,7 @@ fn basic_use() {
            %2 = call i64 @bar(i64 %1)
            ret i64 %2
        }
-    ");
+    ", Some(LIB_WELD_RT));
```

Comment:
You don't need to pass this in the easy_ll tests, do you? isn't it enough to pass None?

Reply:
Fixed.